stock_warehouse_orderpoint_stock_info_unreserved/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.multi
+    def _compute_product_available_qty(self):
+        super(StockWarehouseOrderpoint, self)._compute_product_available_qty()
+        for rec in self:
+            product_available = rec.product_id.with_context(
+                location=rec.location_id.id
+                )._product_available()[rec.product_id.id]
+            rec.product_location_qty_available_not_res = \
+                product_available['qty_available_not_res']
+
+    product_location_qty_available_not_res = fields.Float(
```

Comment:
Fields must be declared on the beginning of the class before methods. https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models

Reply:
Done